explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -418,8 +424,16 @@ object SpectroscopyModesTable {
                                     itcResults.async,
                                     itcProgress.async
                       )
-                }.orEmpty
+                  }
+                  .getOrElse(IO.unit)
```

Comment:
Is the issue with multiple invocations solved this way? Does it have to do with using `.getOrElse(IO.unit)` instead of `.orEmpty`?

Reply:
Upon closer look, it probably has to do with removing `rows` from the dependencies.

Reply:
I think it is about `rows`